### PR TITLE
Development ca - ready for feedback

### PR DESCRIPTION
**Pull request by Littlaa**
Would love some feedback on my project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    </main>
+    <footer class="my-5 pt-5 text-muted text-center text-small container">
+      <p class="mb-1">© 2022 BeSocial</p>
+      <ul class="list-inline">
+        <li class="list-inline-item"><a href="#">Privacy</a></li>
+        <li class="list-inline-item"><a href="#">Terms</a></li>
+        <li class="list-inline-item"><a href="#">Support</a></li>
+      </ul>
+    </footer>
+    <script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
```
Comment (Fermain) [inline index.html:127]:
```suggestion
```

jquery lite is already included in `boostrap.bundle`. You can remove this

File: login.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      </nav>
+    </header>
+    <main>
+      <div class="d-flex justify-content-center mb-3">
+        <h1>LogIn</h1>
+      </div>
+      <div class="d-flex justify-content-center">
+        <div class="tab-content">
+          <div class="tab-pane fade show active" id="pills-login" role="tabpanel" aria-labelledby="tab-login">
+            <form class="d-flex flex-column">
```
Comment (Fermain) [inline login.html]:
```suggestion
            <form class="d-flex flex-column" action="/profile.html">
```

Form should have a working action attribute.

File: login.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                    <input class="form-check-input" type="checkbox" value="" id="loginCheck" checked />
+                    <label class="form-check-label" for="loginCheck"> Remember me </label>
+                  </div>
+                </div>
+
+                <div class="d-flex justify-content-center">
+                  <a href="#!">Forgot password?</a>
+                </div>
+              </div>
+              <button type="submit" formaction="/profile.html" class="btn btn-primary btn-block mb-4 mx-auto">
```
Comment (Fermain) [inline login.html]:
```suggestion
              <button type="submit"  class="btn btn-primary btn-block mb-4 mx-auto">
```

Form action should go on the `<form>` tag. This works and it's not wrong, but it's not what was requested.

File: profile.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    <hr />
+    <footer class="my-5 pt-5 text-muted text-center text-small container">
+      <p class="mb-1">© 2022 BeSocial</p>
+      <ul class="list-inline">
+        <li class="list-inline-item"><a href="#">Privacy</a></li>
+        <li class="list-inline-item"><a href="#">Terms</a></li>
+        <li class="list-inline-item"><a href="#">Support</a></li>
+      </ul>
+    </footer>
+    <script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
```
Comment (Fermain) [inline profile.html:160]:
```suggestion
```

File: src/scss/_index.scss [hunk 5]
```diff
@@ -0,0 +1,22 @@
+hr{
+    border-top: 1px solid #000 !important;
```
Comment (Fermain) [inline src/scss/_index.scss]: `!important` is a sign that Bootstrap is not being overriden properly. It contains styles for `hr` elements that you can override!

File: src/scss/_index.scss [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+.row{
```
Comment (Fermain) [inline src/scss/_index.scss]: You should not change the margins for `row` as these are very important to how bootstrap functions. You can edit the `column gutters` in variables if you want to change how this looks.

File: src/scss/_index.scss [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+    margin-left: 20px; 
+    margin-bottom: 30px;
+}
+
+.head{
+    margin-bottom: 10px;
+}
+
+.user{
+    margin-right: 10px;
```
Comment (Fermain) [inline src/scss/_index.scss]: Look into `margin` utilities for this instead. `.me-2` for example

**Review by Fermain — CHANGES_REQUESTED** [review]
This is a good submission with a few minor mistakes to clean up. You could reduce the number of custom styles you have by making better use of Bootstrap utilities.

File: login.html (hunk 2 above)
Comment (Littlaa) [reply login.html]: Implemented this change.

File: login.html (hunk 3 above)
Comment (Littlaa) [reply login.html]: Implemented this change.

File: index.html (hunk 1 above)
Comment (Littlaa) [reply index.html:127]: When I do this the dropdown to write posts don't work anymore.

File: profile.html (hunk 4 above)
Comment (Littlaa) [reply profile.html:160]: When I do this the dropdown to write posts don't work anymore.

File: src/scss/_index.scss (hunk 5 above)
Comment (Littlaa) [reply src/scss/_index.scss]: Took away the !important and change the color to $pink.

File: src/scss/_index.scss (hunk 6 above)
Comment (Littlaa) [reply src/scss/_index.scss]: Removed these style, found out I didn't need them.

File: src/scss/_index.scss (hunk 7 above)
Comment (Littlaa) [reply src/scss/_index.scss]: Removed these style, found out I didn't need them.

**Pull request merged**